Kai.BaseModule/KaiSDK.cs

```diff
@@ -517,7 +526,12 @@ void FireMagnetometerEvent(MagnetometerEventArgs args)
 
 		private static void DecodeAuthentication(JObject input)
 		{
-			Authenticated = input[Constants.Authenticated].ToObject<bool>();
+			//Authenticated = input[Constants.Authenticated].ToObject<bool>();
```

Comment:
Please revert back to this. The SDK will be updated accordingly. Don't want these temporary work-arounds getting into the mainstream code.